assets/javascripts/overview.js

```diff
@@ -35,5 +35,41 @@ function setupOverview() {
 	    icon.fadeTo('slow', 0.5).fadeTo('slow', 1.0);
 
 	});
+
+    // ensure TODO is false by default because the Browser auto remembers previous state
+    $('#filter-todo').prop('checked', false);
+    
+    // find specified results
+    var varPairs = window.location.search.substring(1).split('&');
```

Comment:
Used the same approach already in other projects.

I searched before whether jQuery provides a convenient way, but only via a plug-in and I decided it is not worth using additional packages for this simple task.